ckan/config/middleware/flask_app.py

```diff
@@ -54,6 +57,13 @@ def make_flask_stack(conf, **app_conf):
     app.template_folder = os.path.join(root, 'templates')
     app.app_ctx_globals_class = CKAN_AppCtxGlobals
     app.url_rule_class = CKAN_Rule
+    extra_paths = config.get('computed_template_paths')
+
+    if extra_paths:
```

Comment:
Shouldn't we use the `ChoiceLoader` always regardless of whether we have `computed_template_paths` in the config at this point?
Otherwise if you start CKAN without any plugin loaded at all you won't detect templates loaded at runtime, like in tests. I know this is impossible right now as we have "core plugins", but wondering if it makes more sense to future proof this.

Reply:
Yeah, it has sense. I prefer to keep things as simple as possible, but here we can allow permanent ChoiceLoader even if there is only one tpl source